x/escrow/handler_test.go

```diff
@@ -725,23 +713,29 @@ func (q query) check(t testing.TB, db weave.ReadOnlyKVStore, qr weave.QueryRoute
 	t.Helper()
 
 	h := qr.Handler(q.path)
-	require.NotNil(t, h, q.path)
+	assert.Equal(t, true, h != nil)
 	mods, err := h.Query(db, q.mod, q.data)
 	if q.isError {
-		require.Error(t, err)
+		assert.Equal(t, true, err != nil)
```

Comment:
Please use `*errors.Error` type instead and of a boolean value and test using `Is` method.